### PR TITLE
Some enhancements... :-)

### PR DESCRIPTION
**Pull request by juliendelplanque**
Modified to allow to modify the output directory. Also rewrited some bash commands in a more compact style (using bash brace expansion) this needed to set the SHELL variable to '/bin/bash' which is ok since it is a common shell.

### CODE REVIEW

**Comment by juliendelplanque** [comment]
Oh, also it is for Makefiles.latex :lollipop:

File: Makefile.latex [hunk 1]
```diff
@@ -6,12 +6,19 @@ SRC_DIR		=
 ##### file to compile, without tex extension;
 NAME		=
 
+##### Output directory
+# If not modified, it is the directory which hold the MakeFile
```
Comment (dannywillems) [inline Makefile.latex]: holds? Or contains? As you want.

**Comment by dannywillems** [comment]
Else it seems OK for me. Maybe improve the `mkdir -p` by using a rule to create it and add it as a dependency of the main rule? It avoids to always execute the command.

File: Makefile.latex (hunk 1 above)
Comment (juliendelplanque) [reply Makefile.latex]: I'll update this.

**Comment by juliendelplanque** [comment]
`mkdir -p` does nothing if nothing has to be done.

**Comment by dannywillems** [comment]
@juliendelplanque Yes of course. It is just to use the power of `make`.

**Pull request merged**